assert and type concepts (#650)
Codecademy/docs: content/python/concepts/built-in-functions/terms/type/type.md

---
Title: 'type()'
Description: 'A function that returns the data type of an object.'
Subjects:

>   - 'Computer Science'

Reviewer: Can we add 'Data Science' as a subject? Same for all the other entries

Tags:
  - 'Built-in Functions'
  - 'Methods'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

A function that returns the data type of an object.

## Syntax

```pseudo
type(some_object)
```

## Example

> The `type()` function can be used to confirm the data type of the object `spam`:

Reviewer: ```suggestion
The `type()` function can be used to confirm the data type of the `spam` object:
```

Author: same as above

Author: @yangc95 made edits and left a couple of comments on your feedback. Should be all set, happy to discuss any points or additional concerns further. Thanks

```py
spam = 10

print(type(spam))

> #Output: int

Reviewer: ```suggestion
# Output: int
```

---
Title: 'type()'

> Description: 'A function that returns the data type of an object.'

Reviewer 2: ```suggestion
Description: 'Returns the data type of an object.'
```

Reviewer 2: Here, since this is a term entry, the `Description` should begin with a verb.

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Built-in Functions'
  - 'Methods'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

A function that returns the data type of an object.

## Syntax

```pseudo
type(some_object)
```

> 

Reviewer 2: Can we include a brief sentence or two that explains what all can be used as `some_object` in `type()`?

Author: I feel like this is self-explanatory. Is there something specific you believe should be addressed here?
